src/App.js

```diff
@@ -1,39 +1,64 @@
 import React, { Component } from 'react'
+import axios from 'axios'
 import Movie from './components/Movie'
+import Header from './components/Header'
 import MovieList from './components/MovieList'
 import SearchBar from './components/SearchBar'
-import movies from './movies.js'
+
+const DEFAULT_STATUS = 'Search a massive database of movies and TV shows!'
+const SearchStatus = ({text}) => (<h1>{text}</h1>)
```

Comment:
You don't need the parenthesis around the `<h1>` tags.

Reply:
done!